src/plugins/vis_type_xy/public/components/detailed_tooltip.test.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { getTooltipData } from './detailed_tooltip';
+import {
+  getAspects,
+  getAspectsWithSplitColumn,
+  getAspectsWithSplitRow,
+  getHeader,
+  getValue,
+} from './detailed_tooltip.mock';
+
+// Get mock data
+const aspects = getAspects();
+const aspectsWithSplitColumn = getAspectsWithSplitColumn();
+const aspectsWithSplitRow = getAspectsWithSplitRow();
+const header = getHeader();
+const value = getValue();
```

Comment:
nit: don't see any reason to return mocked data as a simple functions. It's just plus ~10 of extra lines